systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterIT.java

```diff
@@ -261,8 +259,8 @@ private void assertCmdOption(List<String> cmd, String expectedXmx) {
     }
 
     void assertNoCoErrorsLogged() {
-        //TODO add blacklist for unexpected errors
-        assertThat(kubeClient.searchInLog("deploy", "strimzi-cluster-operator", "60", "Exception", "Error", "Throwable"), isEmptyString());
+        String clusterOperatorLog = kubeClient.searchInLog("deploy", "strimzi-cluster-operator", "60", "Exception", "Error", "Throwable");
```

Comment:
I didn't notice this before, what is "60" ?

Reply:
@ppatierno this is a parameter to get the log for last 60 secs. I made this because sometimes log has a lot of lines.